src/runtime/float.cpp

```diff
@@ -1120,7 +1120,7 @@ static void _addFunc(const char* name, ConcreteCompilerType* rtn_type, void* flo
     v_ff.push_back(BOXED_FLOAT);
     v_fi.push_back(BOXED_FLOAT);
     v_fi.push_back(BOXED_INT);
-    v_fu.push_back(BOXED_FLOAT);
+    v_fu.push_back(UNKNOWN);
```

Comment:
very minor: could you please rename the ```v_fu``` variable to ```v_uu``` now that the first type is unknown